engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetFboWriteMask.java

```diff
@@ -54,70 +49,55 @@
      * @param renderToColorBuffer A boolean indicating whether the Color buffer of the given FBO should be written to.
      * @param renderToDepthBuffer A boolean indicating whether the DepthStencil buffer of the given FBO should be written to.
      * @param renderToLightBuffer A boolean indicating whether the Light Accumulation buffer of the given FBO should be written to.
-     * @param fboName A ResourceUrn identifying the FBO whose render masks have to be modified - usually only the writeOnlyGBuffer FBO.
-     * @param fboManager The FBOManager responsible for managing the given FBO.
+     * @param fbo
      */
-    public SetFboWriteMask(boolean renderToColorBuffer, boolean renderToDepthBuffer, boolean renderToLightBuffer, ResourceUrn fboName, BaseFBOsManager fboManager) {
+    public SetFboWriteMask(boolean renderToColorBuffer, boolean renderToDepthBuffer, boolean renderToLightBuffer, FBO fbo) {
```

Comment:
Ah, when I first created this class, I wanted to put the fboName and fboManager as first parameter, but you asked me to change it :P